docs/checkers/index.md

```diff
@@ -1,13 +1,11 @@
 # PythonTA Checks
 
-This page describes in greater detail the errors that PythonTA checks for.
```

Comment:
There are many change to this file that are not related to the ones you added. You should never reformat files like this, as it makes reviewing your changes much more difficult.

In this specific case, my advice is to take the original file from the master branch, and just add in your changes again.